### PR TITLE
Addressing Issue #531 - Documentation: Components should indicate if they take native props

### LINKED ISSUE #531: Documentation: Components should indicate if they take native props
https://fabricreact.azurewebsites.net/fabric-react/master/#/examples/label#Implementation

Looks really weird for instance and almost looks like it takes no props at all.

### PR DESCRIPTION
**Pull request by oengusmacinog**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #531 
- [x] Include a change request file using `$ npm run change`

#### Description of changes

Added a info MessageBar to indicate when native props can be used on a component:

interfaceexample

### CODE REVIEW

**Comment by Jahnp** [comment]
Gave some feedback in person around the content of the message. This is a great change--really appreciate the clarity this adds.

File: packages/example-app-base/src/components/ComponentPage/ComponentPage.tsx [hunk 1]
```diff
@@ -148,11 +151,45 @@ export class ComponentPage extends React.Component<IComponentPageProps, {}> {
     }
   }
 
+  private _getNativePropsInfo(): JSX.Element | undefined {
+    if (this.props.allowNativeProps) {
+      let elementString: string | string[] | JSX.Element = this.props.nativePropsElement || 'div',
+        componentString: JSX.Element | undefined;
+      // componentString: JSX.Element = <> this component</>;
```
Comment (phkuo) [inline packages/example-app-base/src/components/ComponentPage/ComponentPage.tsx]: remove commented code

File: packages/office-ui-fabric-react/src/components/Pivot/PivotPage.tsx [hunk 2]
```diff
@@ -69,10 +69,12 @@ export class PivotPage extends React.Component<IComponentDemoPageProps, {}> {
             </ExampleCard>
           </div>
         }
+        allowNativeProps={ 'PivitItem' }
```
Comment (phkuo) [inline packages/office-ui-fabric-react/src/components/Pivot/PivotPage.tsx]: spelling

File: packages/example-app-base/src/components/ComponentPage/ComponentPage.tsx [hunk 3]
```diff
@@ -148,11 +151,45 @@ export class ComponentPage extends React.Component<IComponentPageProps, {}> {
...
+            elementString = <>{ elementString } and <code>{ '<' }{ elementArr[_i] }{ '>' }</code></>;
+          }
+        }
+        elementString = <>{ elementString } tags</>;
+      } else {
+        elementString = <><code>{ '<' }{ elementString }{ '>' }</code> tag</>;
+      }
+
+      if (typeof this.props.allowNativeProps === 'string') {
+        // titleString = <> <code>{ this.props.allowNativeProps }</code></>;
```
Comment (phkuo) [inline packages/example-app-base/src/components/ComponentPage/ComponentPage.tsx]: same

**Pull request merged**